packages/decorators/src/tests/memoize.test.ts

```diff
@@ -39,35 +39,6 @@ describe('memoize()', () => {
       expect(myClass.addOne(1)).toEqual(2);
       expect(spy).toHaveBeenCalledTimes(1);
     });
-
-    it('deletes oldest cache item when the cache limit is reached', () => {
```

Comment:
this test is deleted because 1. MAX_MAP_ENTRIES is not being exported 2. this is no longer the functionality of the decorator